publisher/product/ontology/build-about.sh

```diff
@@ -33,7 +25,7 @@ function ontologyCreateAboutFiles () {
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> 
 @prefix owl: <http://www.w3.org/2002/07/owl#> 
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#>
-<${tag_root_url}/LoadFIBOProd> a owl:Ontology;
+<${tag_root_url}/Load${ONTPUB_FAMILY}Prod> a owl:Ontology;
```

Comment:
Should be uppercase:
`<${tag_root_url}/Load${ONTPUB_FAMILY^^}Prod> a owl:Ontology;`